### PR TITLE
fixing ui test instability

### PR DESCRIPTION
**Pull request by kiiadi**

### CODE REVIEW

File: jetbrains-core-gui/build.gradle [hunk 1]
```diff
@@ -40,6 +41,9 @@ task guiTest(type: Test) {
     include '**/*TestSuite*'
 }
 
+// TODO: DISABLED_UI_TEST_ON_2019.3 re-enable when we get compatible guiTestFramework from JB
```
Comment (hunterwerlla) [inline jetbrains-core-gui/build.gradle]: nit: `DISABLED_UI_TEST_ON_2019_3`?

Comment (kiiadi) [reply jetbrains-core-gui/build.gradle]: kewl - can update

Comment (kiiadi) [reply jetbrains-core-gui/build.gradle]: Imma make it `_193`

File: jetbrains-core-gui/build.gradle [hunk 2]
```diff
@@ -41,7 +41,7 @@ task guiTest(type: Test) {
     include '**/*TestSuite*'
 }
 
-// TODO: DISABLED_UI_TEST_ON_2019.3 re-enable when we get compatible guiTestFramework from JB
+// TODO: DISABLED_UI_TEST_ON_193 re-enable when we get compatible guiTestFramework from JB
 guiTest.onlyIf { System.getenv("ALTERNATIVE_IDE_PROFILE_NAME") != "2019.3" }
```
Comment (abrooksv) [inline jetbrains-core-gui/build.gradle]: Using resolveIdeProfileName() == "2019.3" would probably be safer, but eh

Comment (kiiadi) [reply jetbrains-core-gui/build.gradle]: +1

Comment (kiiadi) [reply jetbrains-core-gui/build.gradle]: fixed

Comment (abrooksv) [reply jetbrains-core-gui/build.gradle]: Cool, ship it

**Pull request merged**